app/views/home/index.html.erb

```diff
@@ -10,8 +10,10 @@
     <h2 class="start-large__subtitle"><%= t(:start_page_subtitle) %></h2>
     <div>
       <% if user_signed_in? %>
-        <%= link_to new_project_path do %>
+        <% if current_user.role == 'customer' %>
```

Comment:
home index and projects index have some common rendering code,
for the future, this templates could be extracted to separate view.